# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: gfx/layers/ShareableCanvasRenderer.cpp
# Commit: 05896dbeb434
# Full Hash: 05896dbeb434d6f3ca9324f01eb67020d932f50f
# Author: Jeff Gilbert <jgilbert@mozilla.com>
# Date: 2020-06-15 21:48:38
# Regressor Bug: 1632249
# File Overlap Count: 1
# Description:
#   Bug 1632249 - Determine webgl's layers::TextureType in Present. r=lsalzman
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D79617
# ==============================================================================

diff -r 8439f89e170b -r 05896dbeb434 gfx/layers/ShareableCanvasRenderer.cpp
--- a/gfx/layers/ShareableCanvasRenderer.cpp	Mon Jun 15 18:26:12 2020 +0000
+++ b/gfx/layers/ShareableCanvasRenderer.cpp	Mon Jun 15 18:26:14 2020 +0000
@@ -122,13 +122,8 @@
     }
 
     if (!webgl) return nullptr;
-    if (!forwarder) return nullptr;
 
-    auto texType = layers::TextureType::Unknown;
-    if (forwarder) {
-      texType = layers::PreferredCanvasTextureType(*forwarder);
-    }
-    const auto desc = webgl->GetFrontBuffer(nullptr, texType);
+    const auto desc = webgl->GetFrontBuffer(nullptr);
     if (!desc) return nullptr;
     return GetFrontBufferFromDesc(*desc, flags);
   };